tests/command_line/test_import.py

```diff
@@ -134,6 +134,31 @@ def test_can_import_multiple_sequences(dials_data, tmp_path):
         assert experiment.identifier != ""
 
 
+def test_invert_axis_with_two_sequences_sharing_a_goniometer(dials_data, tmp_path):
+    # Test for regression of https://github.com/dials/dials/issues/2467
+    image_files = sorted(
+        dials_data("centroid_test_data", pathlib=True).glob("centroid*.cbf")
+    )
+    del image_files[4]  # Delete filename to force two sequences
+
+    result = subprocess.run(
```

Comment:
The fact that this does not appear to be recognised by Codecov makes me 🤔

Reply:
I suspect the relevant lines of code were already covered by tests. Adding this just tests behaviour in a particular case, but does not change the coverage.